Microsoft.Toolkit.Uwp.SampleApp/Shell.xaml.cs

```diff
@@ -502,8 +529,8 @@ private void HamburgerMenu_OnOptionsItemClick(object sender, ItemClickEventArgs
                 // If we switch to the Live Preview, then we want to use the Value based Text
                 XamlCodeRenderer.Text = _currentSample.UpdatedXamlCode;
 
-                UpdateXamlRenderAsync(_currentSample.UpdatedXamlCode);
-
+                var t = UpdateXamlRenderAsync(_currentSample.UpdatedXamlCode);
```

Comment:
vat t is unused

Reply:
Why also almost all usages of UpdateXamlRenderAsync is fire and forgot why not awaited ?

Reply:
We don't need the result, it's just to reload the rendering